### PR TITLE
This is the first recipe of MutMap

### PR DESCRIPTION
**Pull request by YuSugihara**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/mutmap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/YuSugihara/MutMap/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  script: python -m pip install --no-deps --ignore-installed .
+  noarch: python
+  number: 2
```
Comment (bgruening) [inline recipes/mutmap/meta.yaml]: this should be 0

File: recipes/mutmap/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+requirements:
+  host:
+    - python >=3.5
+
```
Comment (bgruening) [inline recipes/mutmap/meta.yaml]: please add pip

File: recipes/mutmap/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    - seaborn
+
+test:
+  commands:
+    - mutmap -h
+    - mutplot -h
+
+about:
+  home: https://github.com/YuSugihara/MutMap
+  license: GPL
```
Comment (bgruening) [inline recipes/mutmap/meta.yaml]: please add the license_file

File: recipes/mutmap/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  summary: 'MutMap: pipeline to identify causative mutations responsible for a phenotype'
+
+extra:
+  identifiers:
+    - biotools:mutmap
+    - doi:10.1038/nbt.2095
+  notes:
```
Comment (bgruening) [inline recipes/mutmap/meta.yaml]: not needed if you add license_file, license_family etc in about ...

**Comment by YuSugihara** [comment]
Thanks for your review!
I committed again without checking your review..
I will try again after this test with fixation which you recommend.

**Comment by YuSugihara** [comment]
@bioconda/core
Our package past the test.
Please add our package to bioconda.

Thank you

**Comment by YuSugihara** [comment]
@bioconda/core

Hi bioconda/core,
this is the first recipe for me, and I am not familiar with Github.
My recipe past the test.
To add my recipe to bioconda, should I wait?

I searched my package in bioconda, but there is no my package...

Thank you

File: recipes/mutmap/meta.yaml (hunk 3 above)
Comment (dpryan79) [reply recipes/mutmap/meta.yaml]: In your next release, please package the license with the tarballs (you're technically violating the license you've given your own software).

**Comment by dpryan79** [comment]
@bioconda-bot merge

**Comment by dpryan79** [comment]
@YuSugihara The package will be available in ~20 minutes. As a rule, pull requests need to be merged before packages can be made available.

**Comment by YuSugihara** [comment]
Sorry for that...
I am not familiar with Github.
I will to try to fix it in next time.
Thank you for your comment.

**Pull request merged**